client/index.html.tpl

```diff
@@ -49,22 +49,15 @@
 		<div id="main">
 			<div id="windows">
 				<div id="loading" class="window active">
-					<div class="container">
-						<div class="row">
-							<div class="col-xs-12">
-								<h1 class="title" id="loading-title">The Lounge is loading…</h1>
-							</div>
-							<div class="col-xs-12">
-								<p id="loading-page-message">Loading the app… <a href="http://enable-javascript.com/" target="_blank" rel="noopener">Make sure to have JavaScript enabled.</a></p>
-								<p id="loading-slow">
-									This is taking longer than it should, there might be
-									connectivity issues.
-								</p>
-								<button id="loading-reload" class="btn">Reload page</button>
-								<script async src="js/loading-error-handlers.js"></script>
-							</div>
-						</div>
+					<div id="loading-status-container">
+						<img src="img/logo-vertical-transparent-bg.svg" alt="The Lounge" width="256" height="170">
```

Comment:
I often forget, but we allow server config to specify a custom theme, which is then used on loading screen. When theme is dark, result is pretty horrible:

<img width="420" alt="screen shot 2018-02-28 at 23 42 19" src="https://user-images.githubusercontent.com/113730/36827408-6061e3fe-1ce1-11e8-8e15-5d84696450d5.png">

What about having a second image here ([this one](https://github.com/thelounge/thelounge.github.io/blob/astorije/logos/assets/logos/logo/TL_White%26Yellow_Vertical_logotype_Transparent_Bg/TL_White%26Yellow_Vertical_logotype_Transparent_Bg.svg)) hidden with CSS, then custom themes can hide the normal one and display inverted logo only?

(Alternatively, get rid of server config for theme and only have official theme on page load... :trollface: I know, I know, this idea is not liked 😅)

Reply:
> I know, I know, this idea is not liked

It's not liked because those of us using dark themes don't like a bright flash when loading the lounge. If we were to get custom themes to sync from client to server, then I'd be fine with removing the server theme. Only then.

Also, bear in mind that even if you did remove the server theme, the client theme takes over before this screen goes away sometimes, so this still needs to be dealt with.

Reply:
That was not a real suggestion, just a bit of trolling :D
Real suggestion is the 2 images, but there are probably others (css background?)

Reply:
I know you were trolling, I'm just making sure people know the reasons for it :-P 

As for the actual feature...I'm not sure which I prefer. The hidden image is easiest for theme makers, but also most feels like a hack. Maybe just making both images available on the server and themes can set css background, as you say. Shouldn't be too difficult?

Other option (this is bigger than this PR I think) is we can have a way for themes to set themselves as "light" or "dark" and they can inherit things like this from defaults.

Reply:
We could do some color changing dynamically (e.g. from manifest or background color) as it's `svg`.

Reply:
That could also work.